tests/index.test.ts

```diff
@@ -44,5 +44,6 @@ it('should document all rules', () => {
   existingRules.forEach(rule => {
     expect(README.includes(rule)).toBe(true);
     expect(fs.existsSync(`${root}/docs/rules/${rule}.md`)).toBe(true);
+    expect(rules[rule].meta.docs).toBeDefined();
```

Comment:
I suggest to assert that properties of `docs` are also defined and have specific values (to avoid typos)

Reply:
I think relying on the type system is enough here; it won't compile if there are any missing properties or if they don't have expected values.